mrc/control/controller.py

```diff
@@ -0,0 +1,34 @@
+from threading import Thread
+from typing import Callable
+
+from mrc.configuration.configurator import Configurator
+from mrc.localization.locator import Locator
+
+
+class Controller(Thread):
+    def __init__(self, locator: Locator, drive_to_point: Callable([float, float], None), configurator: Configurator):
+        super().__init__()
+        self.drive_to_point = drive_to_point
+        self.locator = locator
+        self.configurator = configurator
+        self.running = False
+
+    def run(self):
```

Comment:
Shouldn't we allow user to use some kind of their own functionality in there? I mean adding call to some user-defined function in loop.

Reply:
What do you mean

Reply:
In standard approach robots work in a loop (sense -> think -> act -> sense -> ...). Now the only thing a robot can do is drive, it cannot do anything else. But lets let it be like that for now.